src/config.ts

```diff
@@ -76,6 +81,19 @@ function loadTrustProxy(value: string = 'FALSE') {
     return value;
 }
 
+function mapTagsToFilters(tags: string | undefined): Array<TagFilter> {
+    if (tags) {
+        return tags.split(',').map((tag) => {
```

Comment:
instead of using split directly you should reuse `resolveStringToArray()` function, which will allow space between the commas as well.